templates/intro-workspaces/README.ipynb

```diff
@@ -84,9 +84,9 @@
    "source": [
     "### Cloud Storage\n",
     "\n",
-    "Workspace local storage is limited to 1GB, so we recommend only using it to store git repos and smaller files. To persist larger files, you can save data to cloud storage.\n",
+    "Because Workspace local storage is limited to 1GB, use it to store Git repos and smaller files. To persist larger files, you can save data to cloud storage.\n",
```

Comment:
```suggestion
    "Because workspace local storage is limited to 1GB, use it to store Git repos and smaller files. To persist larger files, you can save data to cloud storage.\n",
```

I believe we are standardizing on lowercase for workspace, per previous discussions. It's only capital when introduced in the context of "Anyscale Workspace".